apps/research-service/src/validations/validation.service.ts

```diff
@@ -40,8 +40,8 @@ export class ValidationService {
     });
 
     const criteriaList: ValidationCriteriaDto[] = criteriaData.map(criteria => ({
-      criteriaId: criteria.criteriaId,
-      valid: Boolean(criteria.valid),
+      criteriaId: criteria.criteriaId as CriteriaId,
```

Comment:
Let's change the type of `criteriaId` on the model to be `CriteridId` instead of `string` in order to avoid needing to this now and in the future. Make sure to update the other uses below as well.